src/robusta/integrations/kubernetes/autogenerated/events.py

```diff
@@ -1051,28 +1269,29 @@ def get_subject(self) -> FindingSubject:
             subject_type=FindingSubjectType.from_kind(self.obj.kind),
             namespace=self.obj.metadata.namespace,
             node=KubeObjFindingSubject.get_node_name(self.obj),
+            labels=self.obj.metadata.labels,
+            annotations=self.obj.metadata.annotations,
         )
 
 
-
 KIND_TO_EVENT_CLASS = {
-    'pod': PodChangeEvent,
-    'replicaset': ReplicaSetChangeEvent,
-    'daemonset': DaemonSetChangeEvent,
-    'deployment': DeploymentChangeEvent,
-    'statefulset': StatefulSetChangeEvent,
-    'service': ServiceChangeEvent,
-    'event': EventChangeEvent,
-    'horizontalpodautoscaler': HorizontalPodAutoscalerChangeEvent,
-    'node': NodeChangeEvent,
-    'clusterrole': ClusterRoleChangeEvent,
-    'clusterrolebinding': ClusterRoleBindingChangeEvent,
-    'job': JobChangeEvent,
-    'namespace': NamespaceChangeEvent,
-    'serviceaccount': ServiceAccountChangeEvent,
-    'persistentvolume': PersistentVolumeChangeEvent,
-    'persistentvolumeclaim': PersistentVolumeClaimChangeEvent,
-    'networkpolicy': NetworkPolicyChangeEvent,
-    'configmap': ConfigMapChangeEvent,
-    'ingress': IngressChangeEvent
+    "pod": PodChangeEvent,
+    "replicaset": ReplicaSetChangeEvent,
+    "daemonset": DaemonSetChangeEvent,
+    "deployment": DeploymentChangeEvent,
+    "statefulset": StatefulSetChangeEvent,
+    "service": ServiceChangeEvent,
+    "event": EventChangeEvent,
+    "horizontalpodautoscaler": HorizontalPodAutoscalerChangeEvent,
+    "node": NodeChangeEvent,
+    "clusterrole": ClusterRoleChangeEvent,
+    "clusterrolebinding": ClusterRoleBindingChangeEvent,
+    "job": JobChangeEvent,
+    "namespace": NamespaceChangeEvent,
+    "serviceaccount": ServiceAccountChangeEvent,
+    "persistentvolume": PersistentVolumeChangeEvent,
+    "persistentvolumeclaim": PersistentVolumeClaimChangeEvent,
+    "networkpolicy": NetworkPolicyChangeEvent,
+    "configmap": ConfigMapChangeEvent,
+    "ingress": IngressChangeEvent,
```

Comment:
since we added labels + annotations to all of our events we should verify all of these kubernetes types have the metadata field and its manditory so we don't throw an exception when creating them